b2b-sample/guardio-insurance-sample-application/pages/404.js

```diff
@@ -40,11 +39,12 @@ export default function Custom404() {
 
             <div className={style.errorMainDiv}>
 
-              <Image src={errorImage} width={600} alt="404 image"/>
+              <Image src={errorImage} width={600} alt="404 image" />
 
               <p style={{
                 textAlign: 'center'
-              }}><b>The page your searching seems to be missing.</b> <br />You can go back, or contact our <a>Customer Service</a> team if you need any help</p>
+              }}><b>The page your searching seems to be missing.</b> <br />
+                You can go back, or contact our <a>Customer Service</a> team if you need any help</p>
```

Comment:
Add a `.` at the end of the line

Reply:
Fixed with https://github.com/wso2/samples-is/pull/416/commits/5ffcaa54b0986e1d52996996832e3832ed3d6c9a